server/pages/index.js

```diff
@@ -80,7 +81,11 @@ function generateStaticUrls( request ) {
 		urls[ file.path ] = getUrl( file.path, file.hash );
 	} );
 
-	assets = request.app.get( 'assets' );
+	// vendor dll
+	urls.vendor = urls[ 'vendor.' + config( 'env' ) + '.js' ];
```

Comment:
ah, different env! the env in the jade template is the CALYPSO_ENV while this env is the NODE_ENV. We only need two vendor bundles, one for prod and one for dev, not one for each CALYPSO_ENV.